core/extraction/towns_fund_round_three.py

```diff
@@ -683,8 +681,16 @@ def extract_risks(df_risk: pd.DataFrame, project_lookup: dict, programme_id: str
         "Proximity",
         "RiskOwnerRole",
     ]
-    # TODO: check this drop behaviour. Should we only drop if whole row is empty / null?
-    df_risk_all = drop_empty_rows(df_risk_all, "RiskName")
+    df_risk_all.drop(["Pre-mitigated Raw Total Score", "Post-mitigated Raw Total Score"], axis=1, inplace=True)
+    df_risk_all.columns = risk_columns
+    if not round_four:
```

Comment:
Makes me think that I have added round 3 (suboptimal behaviour to this file in the past, and that I should go back and add this kind of logic.

It does make me a little wary though that this file might become bloated with round specific logic, if rounds start to diverge dramatically 🤔

Reply:
1. re your first point - maybe best to go through and check it.

2, I think that is a legitimate concern. We may need larger architectural change if we keep scaling in this way. I think for now this is better than duplicating the whole function with a slight change but agree it won't scale well. We _should_ be moving away from spreadsheets / this template soon.